src/ontology/mondo-ingest.Makefile

```diff
@@ -72,6 +72,8 @@ $(COMPONENTSDIR)/omim.owl: $(TMPDIR)/omim_relevant_signature.txt | component-dow
 			--update ../sparql/fix-labels-with-brackets.ru \
 			--update ../sparql/fix_illegal_punning_omim.ru \
 			--update ../sparql/exact_syn_from_label.ru \
+			--update ../sparql/convert-OMO_0003000-to-MONDO_ABBREVIATION.ru \
+		remove -T config/properties.txt --select complement --select properties --trim true \
```

Comment:
Why was this line, `remove -T config/properties.txt --select complement --select properties --trim true \` added?

Reply:
NVM, I see what's it is doing now.